app/views/provider_interface/application_choices/show.html.erb

```diff
@@ -28,6 +28,16 @@
   </div>
 </div>
 
+<h2 class="govuk-heading-l govuk-!-margin-top-8" id="course-info">Course</h2>
+
+<%= render SummaryListComponent, rows: [
+  { key: 'Provider', value: @application_choice.provider.name_and_code },
+  { key: 'Course', value: @application_choice.course.name_and_code },
+  { key: 'Cycle', value: @application_choice.course.recruitment_cycle_year },
+  { key: 'Preferred location', value: @application_choice.site.name_and_code },
+  { key: 'Study mode', value: @application_choice.course_option.study_mode.humanize },
+] %>
```

Comment:
:+1: to this partial duplication of the Course Choice component from the Support UI